.travis.yml

```diff
@@ -0,0 +1,41 @@
+---
+sudo: false
+dist: trusty
+language: ruby
+cache: bundler
+before_install:
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
+script:
+  - 'bundle exec rake $CHECK'
+bundler_args: --without system_tests
+rvm:
+  - 2.4.1
+env:
+  global:
+    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="git://github.com/puppetlabs/puppet#master"
```

Comment:
This needs to go into .sync.yaml, otherwise it'll get overwritten next time you run `pdk udpate`

Reply:
yeah... unfortuntately there seems to be a bug in `pdk update` which renders it as:

```diff
 env:
   global:
-    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="git://github.com/puppetlabs/puppet#master"
+    - BEAKER_PUPPET_COLLECTION=puppet5 PUPPET_GEM_VERSION="~> 5.0"
+  matrix:
+    - ["global", ["BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION=\"git://github.com/puppetlabs/puppet#master\""]]
```

Working with @bmjen to understand what's going on.